62.CAD/common.hlsl

```diff
@@ -44,9 +44,10 @@ struct PSInput
 	float4 position : SV_Position;
     [[vk::location(0)]] float4 color : COLOR; 
     [[vk::location(1)]] nointerpolation float4 start_end : COLOR1; 
-    [[vk::location(2)]] nointerpolation uint3 lineWidth_eccentricity_objType : COLOR2; 
+    [[vk::location(2)]] nointerpolation uint4 lineWidth_eccentricity_objType_writeToAlpha : COLOR2; 
 };
 
 [[vk::binding(0,0)]] ConstantBuffer<Globals> globals : register(b0);
 [[vk::binding(1,0)]] StructuredBuffer<DrawObject> drawObjects : register(t0);
+[[vk::binding(2,0)]] RWTexture2D<uint> pseudoStencil : register(u0);
```

Comment:
how do you add the `coherent` GLSL qualifier in HLSL?

Reply:
globallycoherent